pkg/admission/secret_webhook.go

```diff
@@ -173,3 +169,11 @@ func validateKubeConfig(kubeconfig []byte) error {
 	}
 	return clientcmd.ConfirmUsable(*apiConfig, apiConfig.CurrentContext)
 }
+
+func isValidURL(serverURL string) bool {
+	parsed, err := url.Parse(serverURL)
+	if err != nil {
+		return false
+	}
+	return parsed.Scheme == "https" && parsed.Host != ""
+}
```

Comment:
There also is https://github.com/cloudoperators/greenhouse/blob/7d8e2b5734981cbe6af869bb18e6a7a7fda35418/pkg/admission/utils.go#L113-L119 inside the Admission package.

Maybe we can combine them and pass a list of schemes, just so we do not have 95% identical helpers in the same package.
```golang
func isValidURL(url string, schemes ...string) bool {
```